todo.cpp

```
@@ -1,5 +1,164 @@
+				newItem.addCompleteness(comp);
+				theList.push_back(newItem);
+				
+				desc = ""; 
+				comp = "";
+				words = "";
+			}
+			else if(word.compare("Start:") == 0 && words.length() == 0){}
+			else if(word.compare("Completeness:") == 0)
+			{
+				//This sets desc and clears words
+				desc = words;	
+				words=""; 
+			}
+			else
+			{
+				//This appends the word to words
+				words = words + (" " + word);
+			}
+		}
+
+		//adds the last toDo item
+		comp=words;
+		toDo newItem;
+                newItem.addDescription(desc);
+                newItem.addCompleteness(comp);
+		theList.push_back(newItem);
+	}
+
+	std::vector<toDo> theList = {};
+};
+
+int main(int argc, char** argv)
+{
+	string argument = argv[1];
+	//for(int i = 0; i <  argc; ++i)
+	//	cout << argv[i] << "\n";
+
+	toDoList myList; //create instance of list
+	if(argument.compare( "-l") == 0)  //check if -l
+		myList.loadList(argv[2]);
+	else if(argument.compare("-h") == 0)
+	{
+		cout << "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n"
+		     << "||     +	 TODO PROGRAM  +          ||\n"
+		     << "||  +				  +       ||\n"
+		     << "||		 USER MANUAL		  ||\n"
+		     << "||    			 +	          ||\n"
+		     << "||		+	+	        , ||\n"
+		     << "||    ,                                   ||\n"
+		     << "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n"
+		     <<	"		   |vv|		    \n"
+		     << "    ,  ,   ,       |vv|    ,       ,       \n"
+		     << "                   |vv|       ,       ,    \n"
+		     << "  ,    ,      ,    |vv|          ,         \n"
+		     << "    ,      ,       |vv|    ,          ,    \n"
+		     << "    ,   ,          |vv|                    \n"
+		     << "                   |vv|      ,      ,      \n \n\n"
+
+		     << "FLAGS:\n"
+		     << "-h		Displays the program's manual\n"
+		     << "-l		Loads a .txt file and displays its content\n";
+	}
+	else
+	{
+		cout<<"Usage: Loads or saves a todo list from a .txt document. EX: todo -l filename.txt \n";
+		return 0;
```

if the program was run incorrectly, then you should return an error code like -1 or 1. There's a standard list out there, even on CPPReference. Many are unaware of the different codes, but they're actually important.

---

`0` means everything went fine, no errors

---

I actually forgot about that, We'll change it.